.travis.yml

```diff
@@ -5,10 +5,22 @@ python:
     - 3.3
     - 3.4
     - 3.5
+    - 3.6
+    - pypy-5.4.1
+    - pypy3.3-5.2-alpha1
+env:
+    - ENVIRON=py
+    - ENVIRON=flake8,docs
+matrix:
+    exclude:
+        - env: ENVIRON=flake8,docs
+    include:
+        - python: "3.6"
+          env: ENVIRON=flake8,docs
```

Comment:
I believe you can simplify this and drop the global `env`: plus the `matrix: exclude`, just keep the `matrix: include:` bit.

Also, maybe drop the "python: 3.6" from the flake8/docs bits, since tox actually uses python 2.7 for those?